.github/renovate.json

```diff
@@ -10,7 +10,8 @@
     "org.apache.maven:maven-settings-builder",
     "org.apache.maven.resolver:maven-resolver-connector-basic",
     "org.apache.maven.resolver:maven-resolver-transport-file",
-    "org.apache.maven.resolver:maven-resolver-transport-http"
+    "org.apache.maven.resolver:maven-resolver-transport-http",
+    "com.google.inject:guice"
```

Comment:
Better to only allow versions < 7.0 with [`allowedversions`](https://docs.renovatebot.com/configuration-options/#allowedversions) than to block all updates, I think. Otherwise, we won't get PRs for new 6.x versions that we would presumably want to have.

Reply:
Oh right, good catch!